pkg/testrunner/scenario.go

```diff
@@ -147,7 +147,11 @@ func runTestCase(t *testing.T, tc TestCase) bool {
 
 	policyContext := engine.NewPolicyContext().WithPolicy(policy).WithNewResource(*resource)
 
-	er := engine.Mutate(context.TODO(), registryclient.NewOrDie(), policyContext)
+	er := engine.Mutate(
```

Comment:
Do we still have scenario tests? I thought we removed it.

Reply:
Isn't it what powers the CLI ?

Reply:
No, none of the cli tests (`make test-cli`) uses this package.

Reply:
🤔